README.md

```diff
@@ -151,6 +151,8 @@ app.set('view engine', 'jade');
 
 ## Plugins
 
+You can find all of the PostHTML plugins in the [searchable catalog](http://maltsev.github.io/posthtml-plugins/).
```

Comment:
I agree with you. I was thinking about that too. But it sounds weird to me: "Almost all? Why you haven't then collected all plugins?". I'd rather omit the adjective: `You can find the PostHTML plugins ...`. Folks, what do you think?